src/app/options.cpp

```diff
@@ -343,43 +489,84 @@ const QString& CommandLineParameterError::messageForUser() const
     return m_messageForUser;
 }
 
+QString wrapText(const QString &text, int initialIndentation = USAGE_TEXT_COLUMN, int wrapAtColumn = WRAP_AT_COLUMN)
+{
+    QStringList words = text.split(' ');
+    QStringList lines = {words.first()};
+    int currentLineMaxLength = wrapAtColumn - initialIndentation;
+    
+    foreach (const QString &word, words.mid(1)) {
+        if (lines.last().length() + word.length() + 1 < currentLineMaxLength) {
+            lines.last().append(" " + word);
+        }
+        else {
+            lines.append(QString(initialIndentation, ' ') + word);
+            currentLineMaxLength = wrapAtColumn;
+        }
+    }
+    
+    return lines.join("\n");
+}
+
 QString makeUsage(const QString &prgName)
 {
     QString text;
     QTextStream stream(&text, QIODevice::WriteOnly);
+    QString indentation = QString(USAGE_INDENTATION, ' ');
 
     stream << QObject::tr("Usage:") << '\n';
 #ifndef Q_OS_WIN
-    stream << '\t' << prgName << " [options] [(<filename> | <url>)...]" << '\n';
+    stream << indentation << prgName << QLatin1String(" [options] [(<filename> | <url>)...]") << '\n';
```

Comment:
I didn't write the original version of that line, so I don't know. From my perspective I don't see any reason not to include it in Windows, though. If you'd like me to remove the `#ifndef Q_OS_WIN` I can do that.